.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,45 @@
+# Name of this GitHub Actions workflow.
+name: Semgrep
+
+on:
+  push:
+    branches: [ "development", feature/*, hotfix/*, main, release/* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "development", feature/*, hotfix/*, release/* ]
+  # Schedule the CI job (this method uses cron syntax):
+  schedule:
+    - cron: '30 3 * * 2'
```

Comment:
Can you cross check if this time is random? so that we do not have too many jobs running at same time